docs/src/gs-cost.md

```
@@ -0,0 +1,53 @@
+  - CPU memory unit pricing
+  - cost monitoring
+  - dashboard pricing configuration
+---
+
+CruiseKube's provides an estimate on the current cost and saving through CruiseKube. The calculations are right now done with number of constraints and assumptions. This document is to highlight those. 
+
+Right now the cost monitoring is in beta, and we don't perform in-depth monitoring of resources for cost and saving. We don't keep a time-series record of the resources, which can increase the accuracy. We take a heuristic approach for this, documented below. 
+
+# Unit/Resource Pricing
+
+In real world, you pay for the instances/node you have provisioned, and it depends if its on demand, spot, reserved etc. We take a different approach, we estimate the average price for CPU: core/hour and per Memory: GB/hour, and use this unit pricing to calculate the cost and saving. 
```

markdownlint-cli2 (0.21.0)

[warning] Trailing spaces
Expected: 0 or 2; Actual: 1

(MD009, no-trailing-spaces)